pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -31,59 +31,108 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/networking/plugin"
 	"istio.io/istio/pilot/pkg/networking/util"
-	"istio.io/istio/pilot/pkg/proxy/envoy/v1"
 	"istio.io/istio/pkg/log"
 )
 
-// Plugin is a mixer plugin.
-type Plugin struct{}
+type mixerplugin struct{}
+
+type attribute = *mpb.Attributes_AttributeValue
+
+type attributes map[string]attribute
+
+const (
+	// UnknownDestination is set for cases when the destination service cannot be determined.
+	// This is necessary since Mixer scopes config by the destination service.
+	UnknownDestination = "unknown"
+
+	//mixerPortName       = "grpc-mixer"
```

Comment:
is there any need to keep these around, since we are using numerical values now?

Reply:
I'm not sure why we got rid of them in the first place :D I'd prefer to keep named ports.

Reply:
No named ports. We are sticking to numeric values. Named ports has caused a lot of pain.

Reply:
I haven't touched that code. Just want to keep it for historical reasons.